### PR TITLE
feat: add welcome page

### PR DESCRIPTION
**Pull request by alexissmirnov**
the world's most minimalist welcome page

### CODE REVIEW

File: src/pages/welcome.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    font-weight: 800;
+    display: flex;
+    justify-content: center;
+    flex-wrap: wrap;
+  }
+`
+
+export const WelcomePage: React.FC = () => {
+  return (
+    <ThemeProvider theme={localTheme}>
```
Comment (vicrep) [inline src/pages/welcome.tsx]: you can remove themeProvider, this override was specific to react-simple-chatbot

File: src/pages/welcome.tsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import styled, { ThemeProvider } from 'styled-components/macro'
+import { theme } from 'theme'
+import logo from 'images/logo.png'
+
+const localTheme = ({ colors, sizes, fontFamily }: typeof theme) => ({
```
Comment (vicrep) [inline src/pages/welcome.tsx]: you can remove this, also specific to react-simple-chatbot

File: src/pages/welcome.tsx [hunk 3]
```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import styled, { ThemeProvider } from 'styled-components/macro'
```
Comment (vicrep) [inline src/pages/welcome.tsx]: please add a new line between vendor and application imports :)

File: src/pages/welcome.tsx [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+  userFontColor: colors.backgroundLight,
+  fontFamily,
+  colors,
+  sizes
+})
+
+const Wrapper = styled('div')`
+  h1 {
+    color: ${props => props.theme.colors.primary};
+    font-size: calc(${props => props.theme.sizes.buttonText} * 1.5);
```
Comment (vicrep) [inline src/pages/welcome.tsx]: font-family is already globally set for you, so no need to include :)

File: src/pages/welcome.tsx [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+    font-family: ${props => props.theme.fontFamily};
+    padding: calc(${props => props.theme.sizes.buttonText} * 0.75);
+    font-weight: 500;
+    display: flex;
```
Comment (vicrep) [inline src/pages/welcome.tsx]: H1s (and H2s, P, etc) should normally not be flex components, as they're not meant to have any non-inline children

File: src/pages/welcome.tsx [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+
+export const WelcomePage: React.FC = () => {
+  return (
+    <ThemeProvider theme={localTheme}>
+      <Wrapper>
+        <img src={logo} />
+        <h2>COVID-19: What you need to know</h2>
+        <h3>Get accurate personalized information from trusted Canadian medical sources regarding COVID-19.</h3>
+
+        <a href="/">Get Started</a>
```
Comment (vicrep) [inline src/pages/welcome.tsx]: please use `Link` from 'react-router-dom' for this

File: src/pages/welcome.tsx [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+  botBubbleColor: colors.backgroundLight,
+  botFontColor: colors.text,
+  userBubbleColor: colors.secondaryLight,
+  userFontColor: colors.backgroundLight,
+  fontFamily,
+  colors,
+  sizes
+})
+
+const Wrapper = styled('div')`
```
Comment (vicrep) [inline src/pages/welcome.tsx]:
```suggestion
const Wrapper = styled.div`
```

Comment (vicrep) [reply src/pages/welcome.tsx]: also, should be named `Container`

File: src/pages/welcome.tsx (hunk 1 above)
Comment (vicrep) [reply src/pages/welcome.tsx]: theme is already injected for all other components

File: src/pages/welcome.tsx (hunk 5 above)
Comment (vicrep) [reply src/pages/welcome.tsx]: looking at this, you can probably be fine with removing all the display: flex inside text elements

Comment (vicrep) [reply src/pages/welcome.tsx]: if you want centering, then use `text-align: center`

File: src/pages/welcome.tsx (hunk 6 above)
Comment (vicrep) [reply src/pages/welcome.tsx]: https://reacttraining.com/react-router/web/api/Link

Comment (alexissmirnov) [reply src/pages/welcome.tsx]: i don't know how to style it, sorry

**Pull request merged**